$NetBSD: patch-aa,v 1.1.2.2 2007/03/20 15:15:01 salo Exp $

NetBSD 3.0 is missing the extern "C" around the definition of
vsnprintf() in <stdio.h>. In NetBSD 4.99.4, it is fixed.

--- src/lib/WPXString.cpp.orig	2007-03-04 10:44:21.000000000 +0100
+++ src/lib/WPXString.cpp	2007-03-18 21:29:54.000000000 +0100
@@ -24,6 +24,12 @@
  * Corel Corporation or Corel Corporation Limited."
  */
 
+#if defined(__NetBSD__)
+extern "C" {
+#include <stdio.h>
+}
+#endif
+
 #include "WPXString.h"
 #include "libwpd_internal.h"
 
